tests/helpers/create-mobiledoc.js

```diff
@@ -58,8 +60,10 @@ export function createSimpleMobiledoc({sectionName='p', text='hello world', mark
     markups: markups,
     sections: [
       [MARKUP_SECTION_TYPE, sectionName, [
-        [MARKUP_MARKER_TYPE, openedMarkups, closedMarkups, text]]
-      ]
+          [MARKUP_MARKER_TYPE, openedMarkups, closedMarkups, text]
+        ],
+        objectToSortedKVArray(attributes)
```

Comment:
The optional section attribute value should only be in this array if the version of mobiledoc is 0.3.2, if 0.3.1 or earlier it should not be present at all. This will ensure the tests are still testing old 0.3.1 docs, which we really care about continuing to support/test.